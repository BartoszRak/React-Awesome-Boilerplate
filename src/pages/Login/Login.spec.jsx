import React from 'react'
import { shallow } from 'enzyme'

import { Login } from './Login'

const initComponent = overrides => {
  const mockProps = {}
  const mockMethods = {}
  const wrapper = shallow(<Login {...mockProps} {...mockMethods} {...overrides} />)
  return { mockProps, wrapper }
}

describe('Login: ', () => {
  it('renders without crashing', () => {
    const { wrapper } = initComponent()
    expect(wrapper).toBeTruthy()
  })
  it('should render as expected', () => {
    const { wrapper } = initComponent()
    expect(wrapper).toMatchSnapshot()
  })
})
