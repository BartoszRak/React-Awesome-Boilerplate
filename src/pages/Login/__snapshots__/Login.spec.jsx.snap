// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login:  should render as expected 1`] = `
<div>
  Login page
</div>
`;
